### PR TITLE
[1LP][RFR] extdb pod smoke tests

### PR DESCRIPTION
**Pull request by izapolsk**
REQUIRES: https://github.com/ManageIQ/wrapanapi/pull/356

### CODE REVIEW

**Comment by dajoRH** [comment]
Lint report for commit aaacbd55814e0130df93cf9fcb510be6fee74c57:

`cfme/tests/pod/test_appliance_crud.py`:
- :red_circle: Line 148:5: **F841** *local variable 'provision_data' is assigned to but never used*

Please, rectify these issues :smirk: .


**No commit flaws detected**.

*CFME QE Bot*

File: cfme/tests/pod/test_appliance_crud.py [hunk 1]
```diff
@@ -143,6 +178,47 @@ def temp_pod_appliance(appliance, provider, appliance_data, pytestconfig):
...
+            # workaround
+            appliance.is_pod = True
+            stack.push(appliance)
+            holder = config.pluginmanager.get_plugin(PLUGIN_KEY)
+            holder.held_appliance = appliance
+            # workaround, appliance looks ready but api may return errors
+            wait_for(is_api_available, func_args=[appliance], num_sec=30)
+            yield appliance
+            stack.pop()
+    finally:
```
Comment (digitronik) [inline cfme/tests/pod/test_appliance_crud.py]: :100:

File: cfme/tests/pod/test_appliance_crud.py [hunk 2]
```diff
@@ -249,8 +325,7 @@ def test_crud_pod_appliance_ansible_deployment(temp_pod_ansible_appliance, provi
     assert navigate_to(proj, 'Dashboard')
 
 
-@pytest.mark.manual
-def test_crud_pod_appliance_ext_db():
+def test_crud_pod_appliance_ext_db(temp_extdb_pod_appliance, provider, setup_provider):
```
Comment (digitronik) [inline cfme/tests/pod/test_appliance_crud.py]: Optional:
It look like all test required `setup_provider` in this file... not related to this PR but if you like you can move to global mark

Comment (izapolsk) [reply cfme/tests/pod/test_appliance_crud.py]: I'm gonna add more tests. So, I can't predict if new tests will need setup_provider. So, I'd leave it as is.

**Pull request merged**